docs/requirements.md

```diff
@@ -1,51 +1,143 @@
 # Requirements and Recommendations for the Underlying Databases of ScalarDB
 
-This document explains the requirements and recommendations in the underlying databases of ScalarDB to make ScalarDB applications work correctly.
+This document explains the requirements and recommendations in the underlying databases of ScalarDB to make ScalarDB applications work correctly and efficiently.
 
-## Common requirements
+## Requirements
 
-This section describes common requirements for the underlying databases when using ScalarDB.
+ScalarDB requires each underlying database to provide certain capabilities to run transactions and analytics on the databases. This document explains the general requirements and how to configure each database to achieve the requirements.
 
-### Privileges to access the underlying databases
+### General requirements
 
-ScalarDB operates the underlying databases not only for CRUD operations but also for performing operations like creating or altering schemas, tables, or indexes. Thus, ScalarDB basically requires a fully privileged account to access the underlying databases.
+#### Transactions
+{:.no_toc}
+ScalarDB requires each underlying database to provide at least the following capabilities to run transactions on the databases:
 
-## Cassandra or Cassandra-compatible database requirements
+- Linearizable read and conditional mutations (write and delete) on a single database record.
+- Durability of written database records.
+- Ability to store arbitrary data besides application data in each database record.
 
-The following are requirements to make ScalarDB on Cassandra or Cassandra-compatible databases work properly and for storage operations with `LINEARIZABLE` to provide linearizability and for transaction operations with `SERIALIZABLE` to provide strict serializability.
+{% capture notice--info %}
+**Note**
+
+You need to have database accounts that have enough privileges to access the databases through ScalarDB since ScalarDB runs on the underlying databases not only for CRUD operations but also for performing operations like creating or altering schemas, tables, or indexes. ScalarDB basically requires a fully privileged account to access the underlying databases.
```

Comment:
@choplin What do you think about this?

Reply:
Sorry for the late reply. The minimum requirement for privileges for ScalarDB Analytics is weaker than ScalarDB since ScalarDB Analytics does not require write nor schema modification privileges. 

However, we could assume that common users re-use the configuration and accounts of ScalarDB for ScalarDB Analytics as well. So, it would be okay to keep this description here. It would also be worth explaining the configuration and account sharing to the ScalarDB Analytics section.

Reply:
@choplin Thanks.
It seems a general rough requirement, so I moved the note to the bottom of the section.
[5376e04](https://github.com/scalar-labs/scalardb/pull/1511/commits/5376e04ef77503c58ef7a42ae706769338fabb17)

Reply:
Thank you for the answer and for fixing it. LGTM.